scripts/lisk_snapshot.sh

```diff
@@ -163,7 +163,7 @@ dropdb --if-exists "$TARGET_DB_NAME" &> /dev/null
 
 echo -e "\n$(now) Deleting snapshots older then $DAYS_TO_KEEP day(s) in $BACKUP_LOCATION"
 mkdir -p "$BACKUP_LOCATION" &> /dev/null
-find "$BACKUP_LOCATION" -name "${SOURCE_DB_NAME}*.gz" -mtime +"$DAYS_TO_KEEP" -exec rm {} \;
+find "$BACKUP_LOCATION" -name "${SOURCE_DB_NAME}*.gz" -mtime +"$(( DAYS_TO_KEEP - 1 ))" -exec rm {} \;
```

Comment:
If i say days to keep 7. Mtime +7 would keep 7 days and only delete things older than that, ie: 8 days.